src/asm/warning.c

```diff
@@ -80,15 +83,65 @@ static char const *warningFlags[NB_WARNINGS_ALL] = {
 	"obsolete",
 	"shift",
 	"shift-amount",
-	"truncation",
 	"user",
 
+	// Parametric warnings
+	"truncation",
+	"truncation",
+
 	/* Meta warnings */
 	"all",
 	"extra",
-	"everything" /* Especially useful for testing */
+	"everything", /* Especially useful for testing */
+};
+
+// First "char" of each string is how many "internal" warnings each of those maps to
+// Second char is the default if no parameter is given
```

Comment:
This comment is no longer accurate or needed, since the struct field names are self-explanatory.
```suggestion
```